fire/core.py

```diff
@@ -681,64 +683,72 @@ def _ParseKeywordArgs(args, fn_args, fn_keywords):
     fn_keywords: The argument name for **kwargs, or None if **kwargs not used
   Returns:
     kwargs: A dictionary mapping keywords to values.
+    remaining_kwargs: A list of the unused kwargs from the original args.
     remaining_args: A list of the unused arguments from the original args.
   """
   kwargs = {}
-  if args:
-    remaining_args = []
-    skip_argument = False
-
-    for index, argument in enumerate(args):
-      if skip_argument:
-        skip_argument = False
-        continue
-
-      arg_consumed = False
-      if argument.startswith('--'):
-        # This is a named argument; get its value from this arg or the next.
-        got_argument = False
-
-        keyword = argument[2:]
-        contains_equals = '=' in keyword
-        is_bool_syntax = (
-            not contains_equals and
-            (index + 1 == len(args) or args[index + 1].startswith('--')))
-        if contains_equals:
-          keyword, value = keyword.split('=', 1)
-          got_argument = True
-        elif is_bool_syntax:
-          # Since there's no next arg or the next arg is a Flag, we consider
-          # this flag to be a boolean.
-          got_argument = True
-          if keyword in fn_args:
-            value = 'True'
-          elif keyword.startswith('no'):
-            keyword = keyword[2:]
-            value = 'False'
-          else:
-            value = 'True'
+  remaining_kwargs = []
+  remaining_args = []
+
+  if not args:
```

Comment:
I add a return statement here because pylint complains about "Too many nested blocks (6/5)". Not sure if I should do it this way.